gitgud/levels/util.py

```diff
@@ -193,42 +193,51 @@ def test_level(level, test):
     return True
 
 class NamedList:
-    # Pass in list of tuples in format (reference, Object), where reference is of type str
-    def __init__(self, data=[]):
-        self.namedict = {d[0]:i for i, d in enumerate(data)}
-        self.items = [obj for name, obj in data]
+    # names is a list populated with type str, items is a list populated with any type 
+    def __init__(self, names, items):
+        assert len(names) == len(items)
+        self._name_dict = {name:index for index, name in enumerate(names)}
+        self._items = [item for name, item in zip(names, items)]
     
     def __getitem__(self, query):
         if isinstance(query, int):
-            return self.items[query]
+            return self._items[query]
         elif isinstance(query, str):
-            return self.items[self.namedict[query]]
+            return self._items[self._name_dict[query]]
         else:
             return None
```

Comment:
You should change this to `raise ValueError('Level not found')`. When things fail, we should raise an exception unless we for some reason intend on using that `None` value